Task2/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="css/content.css" rel="stylesheet" type="text/css">
+    <meta charset="UTF-8">
+    <title>Task2</title>
+</head>
+<body>
+<h1 id="b">Chapters</h1>
+    <nav>
+        <a href="#a1">What is Lorem ipsum</a> |
+        <a href="#a2">Why do we us it</a> |
+        <a href="#a3">Where does it come from?</a> |
+        <a href="#a4">Where can I get some?</a> |
+        <a href="#a5">There standards</a> |
+        <a href="#a6">Section</a> |
+        <a href="#a7">1914</a> |
+        <a href="#a8">Section</a> |
+        <a href="#a10">Donate</a> |
+        <a href="https://www.google.com" target="_blank">GOOGLE</a>
+    </nav>
```

Comment:
Don't use ` |` as hardcoded element for adding divider between links. Use for it CSS power e.g. pseudo-elements. Read more [here](https://stackoverflow.com/questions/1936661/vertical-dividers-on-horizontal-ul-menu) and [here](https://coderwall.com/p/cp3jza/pseudo-class-css-menu-divider)

Reply:
Done